airgun/entities/host.py

```diff
@@ -308,7 +308,7 @@ class ShowAllHosts(NavigateStep):
 
     @retry_navigation
     def step(self, *args, **kwargs):
-        self.view.menu.select('Hosts', 'All Hosts')
+        self.view.menu.select('Hosts', 'Hosts', 'All Hosts')
```

Comment:
I don't think it's needed to have 'Hosts' twice here (same for 'Configure'), as 'All Hosts' is directly under 'Hosts', only 2 levels down

Reply:
yes, I know it might be skipped intensionally earlier, but to keep this pattern of paths consistent thought better to have this, do you think it could fail the tests?

Reply:
@Gauravtalreja1 can you please test this?

Reply:
Probably will not cause failures, but I feel the additional step in navigation adds unnecessary delay to tests as these navigation routes are used rather frequently.

Reply:
@mstrlc yes, it shouldn't cause any failures, and I don't think it should add any significant delay but considering the entire testsuite that's good point, will revert this changes

Reply:
Updated, thank you